MainModule/Client/UI/Default/HelpButton.lua

```diff
@@ -12,6 +12,12 @@ return function(data, env)
 	end
 
 	local UI = client.UI
+	local Functions = client.Functions
+
+	if Functions.CheckDate(1, 4) or Functions.Checkdate(31, 10) then
+		UI.MakeGui("Effect", {Mode = "lifeoftheparty";})
+		return
```

Comment:
Remove the return statement.

We shouldn't fully disable the help button if this was to be added